cmd/server/flags.go

```diff
@@ -161,6 +161,16 @@ var flags = []cli.Flag{
 		Name:    "gating-service",
 		Usage:   "gated build endpoint",
 	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_YAML_ENDPOINT"},
+		Name:    "configuration-service",
+		Usage:   "configuration service endpoint",
+	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_YAML_SECRET"},
+		Name:    "configuration-service-secret",
+		Usage:   "configuration service secret",
+	},
```

Comment:
The `EnvVars` and `Name` are different, could using the similar names be better?

Most other EnvVars and Names are using similar names

Reply:
@wxiaoguang do you have any recommendations ?
Eg:
WOODPECKER_CONFIG_SERVICE_ENDPOINT
WOODPECKER_CONFIG_SERVICE_SECRET

?

Reply:
WOODPECKER_CONFIG_SERVICE_ENDPOINT <=> --config-service-endpoint
WOODPECKER_CONFIG_SERVICE_SECRET <=> --config-service-secret

It sounds very accurate 😊